content/WebAuthn/WebAuthn_Developer_Guide/WebAuthn_Client_Registration.adoc

```diff
@@ -68,19 +68,23 @@ _Note: In the WebAuthn API, all binary parameters (`user.id`, `challenge` etc.)
 
 **user:** contains data about the user account. The **displayName** provides a user-friendly name which may be displayed to the user. The **name** attribute takes a traditional username or the like, to disambiguate accounts with similar **displayNames**. The **id** attribute is the user handle that is generated by the RP and used to uniquely identify a user. Unlike **name**, **id** is not meant to be human-readable and should not contain personally identifying information such as an e-mail address.
 
-**2.** The JavaScript client calls `navigator.credentials.create()`. The browser validates the `rp.id` against the origin, hashes the `clientData`, and calls `authenticatorMakeCredential` method.
+**Step 2:** The JavaScript client calls `navigator.credentials.create()`. The browser validates the `rp.id` against the origin, hashes the `clientData`, and calls `authenticatorMakeCredential` method.
 
-**3.** Before proceeding, the authenticator will ask for some form of user consent. After verifying consent, the authenticator will create a new asymmetric key pair and safely store the private key. The public key becomes part of the attestation, which the authenticator signs over with the attestation private key. The manufacturer’s attestation certificate chain may also be returned so that the relying party can validate the device back to a root of trust.
+_Note: `MakeCredential` will make new credentials every time unless you pass in the list of previously created credentials in the exclude list. See `excludeCredentials` above._
 
-**4.** The new public key, a credential id, and other attestation data are returned to the browser where they become the attestationObject. 
+**Step 3:** Before proceeding, the authenticator will ask for some form of user consent. After verifying consent, the authenticator will create a new asymmetric key pair and safely store the private key. The public key becomes part of the attestation, which the authenticator signs over with the attestation private key. The manufacturer’s attestation certificate chain may also be returned so that the relying party can validate the device back to a root of trust.
 
-**5.** The `navigator.credentials.create()` Promise resolves to a `PublicKeyCredential` which returned to the RP to finalize the registration.
+**Step 4:** The new public key, a credential id, and other attestation data are returned to the browser where they become the attestationObject. 
+
+**Step 5:** The `navigator.credentials.create()` Promise resolves to a `PublicKeyCredential` which returned to the RP to finalize the registration. 
+
+_Note: When you retrieve the public key credentials, even if you use the same object to generate the registration, the value of the credential returned will be different each time, unless you pass in the list of previously created credentials. See Note to Step 2 above._
```

Comment:
Isn't this just repeating what's already noted in step 2?

Reply:
Yes, it is - but someone STILL succeeded in 'overlooking' the note in step 2. So this time I've removed the note to step 5 and added some of the additional information that was in it to the note in step 2.

Reply:
Ok, I see.